Tests/NimbleTests/Matchers/BeEmptyTest.swift

```diff
@@ -73,5 +69,35 @@ final class BeEmptyTest: XCTestCase, XCTestCaseProvider {
         failsWithErrorMessage("expected to be empty, got <foo>") {
             expect("foo").to(beEmpty())
         }
+
+        failsWithErrorMessage("expected to not be empty, got <NSSortOptions(rawValue: 0)>") {
```

Comment:
Having our own option set type here would also decouple us from `NSSortOptions`'s implementation of string conversion.